test/spec/modules/id5IdSystem_spec.js

```diff
@@ -19,7 +19,7 @@ import * as utils from 'src/utils.js';
 
 let expect = require('chai').expect;
 
-describe('ID5 ID System', function() {
+describe.only('ID5 ID System', function() {
```

Comment:
I think you forgot to get rid of `.only`, because of this all test cases are not running.

Reply:
whoops my bad...fixed now